packages/models/src/risk-analysis/riskAnalysis.ts

```diff
@@ -53,3 +55,55 @@ export const RiskAnalysis = z.object({
   createdAt: z.coerce.date(),
 });
 export type RiskAnalysis = z.infer<typeof RiskAnalysis>;
+
+export const RiskAnalysisTemplateAnswerAnnotationDocument = z.object({
```

Comment:
I suggest to place those types definitions in a different files
for example `packages/models/src/risk-analysis/riskAnalysisTemplate.ts`, to avoid possible ambiguity?